### PR TITLE
Add job for ceph-iscsi-* stable releases

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: ceph-iscsi-stable/config/definitions/ceph-iscsi-stable.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    node: 'centos7 && x86_64 && huge'
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph-iscsi-stable'
+    concurrent: true
+    parameters:
+      - string:
+          name: CEPH_ISCSI_CONFIG_BRANCH
+          description: "The git branch (or tag) to build"
+          default: "2.3"
```
Comment (dillaman) [inline ceph-iscsi-stable/config/definitions/ceph-iscsi-stable.yml]: 2.6

File: ceph-iscsi-stable/build/build_rpm [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+done
+
+cd $WORKSPACE
+
+# All three projects should share the same major version so we'll use that
+VERSION=$(echo $VERSION | cut -d '.' -f1)
+# The VERSION and GIT_COMMIT aren't really important here.  We just feed it the last project's info so the CI works.
+chacra_endpoint="ceph-iscsi/${VERSION}/${GIT_COMMIT}/${DISTRO}/${RELEASE}"
+chacra_repo_endpoint="${chacra_endpoint}/flavors/default"
+RPM_RELEASE=`grep Release $WORKSPACE/ceph-iscsi-tools/ceph-iscsi-tools.spec | sed 's/Release:[ \t]*//g' | cut -d '%' -f 1`
```
Comment (dillaman) [inline ceph-iscsi-stable/build/build_rpm]: Nit: perhaps use the ceph-iscsi-config version? The tools RPM is rarely touched.

Comment (djgalloway) [reply ceph-iscsi-stable/build/build_rpm]: Those variables (RPM_RELEASE, RPM_VERSION, and PACKAGE_MANAGER_VERSION) don't actually get used anywhere so I just removed those three lines.

File: ceph-iscsi-stable/config/definitions/ceph-iscsi-stable.yml (hunk 1 above)
Comment (djgalloway) [reply ceph-iscsi-stable/config/definitions/ceph-iscsi-stable.yml]: Thanks. Fixed!

File: ceph-iscsi-stable/build/build_rpm (hunk 2 above)
Comment (dillaman) [reply ceph-iscsi-stable/build/build_rpm]: But what defines the ```RELEASE``` environment variable? It seems like perhaps it's the last package to be built, but is the build order fixed?

Comment (djgalloway) [reply ceph-iscsi-stable/build/build_rpm]: https://github.com/ceph/ceph-build/pull/1105/commits/0424f2ed27ea4e8bf227f976c375793bca16c8f9#diff-429e473536d6eb85857849642ff4702aR15

`RELEASE` will always be '7'

Comment (dillaman) [reply ceph-iscsi-stable/build/build_rpm]: Ah -- d'oh.

**Comment by djgalloway** [comment]
http://tracker.ceph.com/issues/22143

**Pull request merged**